base/src/main/java/io/spine/code/proto/ref/EnrichmentForOption.java

```diff
@@ -22,26 +22,27 @@
 
 import com.google.common.base.Splitter;
 import com.google.common.collect.ImmutableList;
+import com.google.protobuf.DescriptorProtos;
 import com.google.protobuf.DescriptorProtos.DescriptorProto;
+import io.spine.code.proto.MessageOption;
+import io.spine.option.OptionsProto;
 
 import java.util.Optional;
 
-import static io.spine.option.Options.option;
-import static io.spine.option.OptionsProto.enrichmentFor;
-
 /**
  * Parses {@code (enrichment_for)} option value of an enrichment type definition.
  *
  * <p>The option may have one or more reference to a type separated with commas.
  */
-public final class EnrichmentForOption {
+public final class EnrichmentForOption extends MessageOption<String> {
```

Comment:
It's not one string. It refers to a number of type references. So, we basically need:

1. An abstract base that can parse strings, split them, and have an abstract factory method for creating objects by those strings.
2. Implement this abstract base for `EnrichmentForOption`, teaching it to create `TypeRef`s from the strings.